File: Code/HVIT/HVIT_JAVA/src/JavaPractice/JavaOOPBasic/SoPhuc/SoPhuc.java
package JavaPractice.JavaOOPBasic.SoPhuc;

import java.util.Scanner;

public class SoPhuc {
    private double PhanThuc;
    private double PhanAo;

    public double getPhanThuc() {
        return PhanThuc;
    }

    public void setPhanThuc(double phanThuc) {
        PhanThuc = phanThuc;
    }

    public double getPhanAo() {
        return PhanAo;
    }

    public void setPhanAo(double phanAo) {
        PhanAo = phanAo;
    }

    public SoPhuc() {
        Scanner sc = new Scanner(System.in);
        System.out.printf("Nhap phan thuc: ");
        setPhanThuc(sc.nextDouble());
        System.out.printf("Nhap phan ao: ");
        setPhanAo(sc.nextDouble());
    }

    public SoPhuc(double PhanThuc, double PhanAo) {
        setPhanAo(PhanThuc);
        setPhanAo(PhanAo);
    }

    public void InThongTin() {
        System.out.println(getPhanThuc() + "+" + getPhanAo() + "i");
    }
}

File: Code/HVIT/HVIT_EX/HVIT_OOP_TEST/OOP/SoPhuc/SoPhuc.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace SoPhuc
{
    class SoPhuc
    {
        public int PhanThuc { get; set; }
        public int PhanAo { get; set; }
        public SoPhuc()
        {
            Console.Write("Nhap phan thuc: ");
            PhanThuc = int.Parse(Console.ReadLine());
            Console.Write("Nhap phan ao: ");
            PhanAo = int.Parse(Console.ReadLine());
        }
        public SoPhuc(int PhanThuc, int PhanAo)
        {
            this.PhanAo = PhanAo;
            this.PhanThuc = PhanThuc;
        }
        public void HienThi()
        {
            Console.WriteLine($"{PhanThuc} + {PhanAo}i");
        }
    }
}
